From 0e31067519bb00a158d60adfd424d09fd4c37283 Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Wed, 4 Apr 2018 17:17:15 +0800
Subject: [PATCH 3569/5242] MLK-17981[IMX8QXP B0] Change mek dts to enable vpu
 B0 encoder and decoder

commit  728c6a384c89a79f4712474c88ad372206e50564 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change fsl-imx8qxp-mek.dts file to enable vpu B0 encoder and decoder for
IMX8QXP B0 mek board

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 0b286d2..a2d0b53 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -905,6 +905,14 @@
 };
 
 &vpu {
+	status = "disabled";
+};
+
+&vpu_decoder {
+	status = "okay";
+};
+
+&vpu_encoder {
 	status = "okay";
 };
 
-- 
1.7.9.5

